src/components/tag/styles.ts

```diff
@@ -0,0 +1,34 @@
+import { StyleSheet } from 'react-native';
+import { AppColor, TagType } from 'src/common/enums';
+
+const styles = StyleSheet.create({
+  global: {
+    borderRadius: 15,
+    paddingVertical: 1,
+    paddingHorizontal: 10,
+    flexDirection: 'row',
+    alignItems: 'center',
+  },
+  [TagType.COMMON]: {
+    backgroundColor: AppColor.TAG_COMMON_BACKGROUND,
+  },
+  [TagType.ATTENTION]: {
+    backgroundColor: AppColor.TAG_ATTENTION_BACKGROUND,
+    borderWidth: 2,
+    borderColor: AppColor.ACCENT,
+  },
+  text: {
+    color: AppColor.SECONDARY,
+    opacity: 0.6,
+    fontFamily: 'NunitoSans-Bold',
```

Comment:
Just `fontWeight: 700`

Reply:
I'll change it for headers, but for tags fontWieght: 700 has different from Figma style, fontWieght: 500/600 doesn't work in Android